### PR TITLE
Easier run the tests, update the documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/server_side.rst.mako [hunk 1]
```diff
@@ -9,107 +9,100 @@ Create development environment in a project
...
 
-      export ADD_JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
+Docker images dependencies::
 
+.. image:: docker.png
+.. source file is docker.dia.
 
-Tests
------
+Image nomoclature:
```
Comment (arnaud-morvan) [inline doc/developer/server_side.rst.mako]: nom**o**clature

File: doc/developer/server_side.rst.mako [hunk 2]
```diff
@@ -9,107 +9,100 @@ Create development environment in a project
...
 
-Running tests
-~~~~~~~~~~~~~
-
-To be able to run c2cgeoportal tests you need to have the c2cgeoportal source
-code, and a make environment for it. So do that first, as described below.
+* ``camptocamp/geomapfish-*`` for all images generated by/for c2cgeoportal
+* ``camptocamp/geomapfish-test-*`` for all images used only for the CI (not pushed on a Docker repository)
+* ``camptocamp/testgeomapfish-*`` the images related to the generated application,
+        will be ``camptocamp/<package>-*`` in the client application.
```
Comment (arnaud-morvan) [inline doc/developer/server_side.rst.mako]: I do not understand the last one. Do you talk about scaffold generated client app images ?

File: doc/developer/server_side.rst.mako [hunk 3]
```diff
@@ -9,107 +9,100 @@ Create development environment in a project
...
 
-Install c2cgeportal from source
-...............................
+Unit tests
+..........
 
-Check out c2cgeoportal from GitHub:
+To run the unit tests do this, buils all the dependencies:
```
Comment (arnaud-morvan) [inline doc/developer/server_side.rst.mako]: buil**d**s

"do this" is not useful for me.

I would prefer : "Before to run the tests, install and build all dependencies :"

File: docker/build/Dockerfile [hunk 4]
```diff
@@ -26,6 +26,7 @@ RUN \
 
 COPY run /usr/bin/
 COPY sudoers /etc/
+COPY bin/* /usr/bin/
```
Comment (arnaud-morvan) [inline docker/build/Dockerfile]: I do not see where this bin forlder come from ?

File: doc/developer/server_side.rst.mako (hunk 1 above)
Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: ? It's what I write...

File: doc/developer/server_side.rst.mako (hunk 2 above)
Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: Is is clearer like this:
```
* ``camptocamp/<package>-*`` the images of the client application, will be ``camptocamp/testgeomapfish-*``
    in the continues integration chain.
```

File: doc/developer/server_side.rst.mako (hunk 3 above)
Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: +1, thanks :-)

File: docker/build/Dockerfile (hunk 4 above)
Comment (sbrunner) [reply docker/build/Dockerfile]: From here: https://github.com/camptocamp/c2cgeoportal/tree/ci-doc/docker/build/bin

File: doc/developer/server_side.rst.mako (hunk 1 above)
Comment (arnaud-morvan) [reply doc/developer/server_side.rst.mako]: yes, but in english, like in french, it is nom**en**clature

Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: ok :-)

File: doc/developer/server_side.rst.mako (hunk 2 above)
Comment (arnaud-morvan) [reply doc/developer/server_side.rst.mako]: When I read the text from the PR a second time it seems clear now. Sorry for the noise.

Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: The second one isn't clearer? For me yes...

Comment (arnaud-morvan) [reply doc/developer/server_side.rst.mako]: or ``camptocamp/testgeomapfish-*`` for all images used by the CI to test applications generated using scaffolds.

Comment (arnaud-morvan) [reply doc/developer/server_side.rst.mako]: or to test scaffolds used to generate applications.

Comment (sbrunner) [reply doc/developer/server_side.rst.mako]: +1 updated :-)

**Comment by sbrunner** [comment]
Thanks for the review @arnaud-morvan

**Pull request merged**